enterprise/cmd/frontend/internal/authz/resolvers/repositories.go

```diff
@@ -109,7 +111,7 @@ func (r *repositoryConnectionResolver) Nodes(ctx context.Context) ([]*graphqlbac
 	}
 	resolvers := make([]*graphqlbackend.RepositoryResolver, len(repos))
 	for i := range repos {
-		resolvers[i] = graphqlbackend.NewRepositoryResolver(r.db, repos[i])
+		resolvers[i] = graphqlbackend.NewRepositoryResolver(log.Scoped("Nodes", ""), r.db, repos[i])
```

Comment:
```suggestion
		resolvers[i] = graphqlbackend.NewRepositoryResolver(log.Scoped("repos.nodes", ""), r.db, repos[i])
```

The "nodes" scope doesn't make much sense here, because it's a top level scope, as it's not tied to a particular resolver.